### DISCUSSION TITLE
How to use a single process for the whole app?

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2403] [Q&A]
I noticed that there are 3 different processes being launched (one for the settings, one for file browser, and one for editor).

This is a little bit confusing when task switching on Android, as most of the time I have 2 open instances of the app (the file browser and the editor).

Is it possible to use a single process for all 3? And how can I do this change? I would appreciate any suggestions.

**Comment by harshad1** [comment]
Getting this to work right has been a bit of a problem. Will continue to iterate

**Reply by ghost** [reply]
I just made this an issue https://github.com/gsantner/markor/issues/2410
